examples/Basics/simple_flows_and_runs_tutorial.py

```diff
@@ -48,6 +48,13 @@
 clf = KNeighborsClassifier(**knn_parameters)
 clf.fit(X_train, y_train)
 
+# Option A: auto-publish the estimator via unified helper (requires openml-sklearn extension).
+try:
```

Comment:
try/except for case distinction is not good. This does not allow to distinguish the intended case from a genuine failure in the "try" branch.

Plus, in the examples, there should be no try/except.